### PR TITLE
Milestone: Add request spec files for all controllers

### PR DESCRIPTION
**Pull request by geekelo**
Add request spec files for the user and posts controller to test for
- [x]  If the response status was correct.
- [x] If a correct template was rendered.
- [x] If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      get user_posts_path(user)
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get user_posts_path(user)
+      expect(response.body).to include('post index')
+    end
+  end
+end
```
Comment (OyePriscilla) [inline spec/requests/post_spec.rb]: - [ ] Please kindly implement the missing test for the `post_show` controller as well in accordance to the project requirement. ℹ️ 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @geekelo 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Used RSpec gem for testing ✔️ 
Create a Request spec file for `users_index, users_show and posts_index` your controllers ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/post_spec.rb (hunk 1 above)
Comment (geekelo) [reply spec/requests/post_spec.rb]: @OyePriscilla Thank you
Kindly do a favor by giving a star. Thanks in advance

**Review by addod19 — APPROVED** [review]
## Hi @geekelo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Nice job adding the required test case :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by geekelo** [comment]
@addod19 Thank you
Kindly do a favor by giving a star. Thanks in advance

**Pull request merged**